crates/connectors/ndc-postgres/src/configuration/version2.sql

```diff
@@ -102,11 +102,17 @@ WITH
       att.attnum AS column_number,
       att.atttypid AS type_id,
       CASE WHEN att.attnotnull THEN 'nonNullable' ELSE 'nullable' END
-      AS nullable
-      -- Columns that will likely be of interest soon:
-      -- attidentity
-      -- attgenerated
-      -- atthasdef
+      AS nullable,
+      CASE WHEN att.atthasdef THEN 'hasDefault' ELSE 'noDefault' END
+      AS has_default,
+      CASE WHEN att.attidentity = 'd' THEN 'identityByDefault'
+           WHEN att.attidentity = 'a' THEN 'identityAlways'
+           ELSE 'notIdentity'
+      END
+      AS is_identity
+      -- skipped because yugabyte is based on pg11 that does not have this field.
+      -- CASE WHEN att.attgenerated = 's' THEN 'isGenerated' ELSE 'notGenerated' END
+      -- AS is_generated
```

Comment:
There is a trick being employed in `pg_table_metadata.sql` of HGE v2 which deals with this very issue. It could well be worth it to port that over.

Reply:
Ah good to know, for reference: https://github.com/hasura/graphql-engine-mono/blob/main/server/src-rsr/pg_table_metadata.sql#L70-75

Will do that in a follow-up PR.